media-tv/mythtv/files/mythtv-Fix_create_webbrowser_window.patch

```diff
@@ -0,0 +1,58 @@
+--- a/libs/libmythui/mythuiwebbrowser.cpp
++++ b/libs/libmythui/mythuiwebbrowser.cpp
+@@ -873,4 +873,8 @@
+ void MythUIWebBrowser::Init(void)
+ {
```

Comment:
missing patch header. please use git format-patch style if you took it from upstream, otherwise add description manually.

Reply:
Added it.